File: resourcespoet/src/test/java/com/commit451/resourcespoet/BoolTest.java
package com.commit451.resourcespoet;

import org.junit.Test;

/**
 * Tests the resources creation
 */
public class BoolTest {

    @Test
    public void boolTest() throws Exception {
        ResourcesPoet poet = ResourcesPoet.create()
                .addBool("is_cool", true);

        TestUtil.assertEquals("bool.xml", poet);
    }
}

File: src/test/kotlin/com/commit451/resourcespoet/BoolTest.kt
package com.commit451.resourcespoet

import org.junit.Test

/**
 * Tests the resources creation
 */
class BoolTest {

    @Test
    fun boolTest() {
        val poet = ResourcesPoet.create()
            .addBool("is_cool", true)

        TestUtil.assertEquals("bool.xml", poet)
    }
}
